.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since we've added `.idea/` to the gitignore, it's strange that this is showing up? Could we remove it?